docs/src/developing/plugins/geyser-plugins.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Plugins
+title: Geyser Plugins
 ---
 
 ## Overview
 
 Validators under heavy RPC loads, such as when serving getProgramAccounts calls,
 can fall behind the network. To solve this problem, the validator has been
 enhanced to support a plugin mechanism through which the information about
-accounts and slots can be transmitted to external data stores such as relational
-databases, NoSQL databases or Kafka. RPC services then can be developed to
-consume data from these external data stores with the possibility of more
-flexible and targeted optimizations such as caching and indexing. This allows
-the validator to focus on processing transactions without being slowed down by
-busy RPC requests.
+accounts, slots, blocks, and transactions can be transmitted to external data
+stores such as relational databases, NoSQL databases or Kafka. RPC services then
+can be developed to consume data from these external data stores with the
+possibility of more flexible and targeted optimizations such as caching and
+indexing. This allows the validator to focus on processing transactions without
+being slowed down by busy RPC requests.
```

Comment:
Maybe we should update it here with the Geyer plugin name. For example 'support a plugin mechanism named Geyer Plugins'..